src/write-ui-env-file.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+import path, { dirname } from "path";
+import { promises as fs } from "fs";
+import { fileURLToPath } from "url";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const outputPath = path.join(__dirname, "../../services/ui-src", "public");
+const configFilePath = path.resolve(outputPath, "env-config.js");
+
+const region = "us-east-1";
+
+export async function writeLocalUiEnvFile(apiUrl: string) {
+  console.log("apiUrlapiUrlapiUrlapiUrlapiUrlapiUrl");
+  console.log(apiUrl);
+  const envVariables = {
+    SKIP_PREFLIGHT_CHECK: "true",
+    STAGE: "localhost",
```

Comment:
with seds I believe we originally used stage localstack (then removed it?) not localhost.